openslides/users/config_variables.py

```diff
@@ -90,3 +90,42 @@ def get_config_variables():
         weight=570,
         group='Participants',
         subgroup='PDF')
+
+    # Email
+
+    yield ConfigVariable(
+        name='users_email_sender',
+        default_value='noreply@yourdomain.com',
+        input_type='string',
+        label='Email sender',
+        weight=600,
+        group='Participants',
+        subgroup='Email')
+
+    yield ConfigVariable(
+        name='users_email_subject',
+        default_value='Your login for {event_name}',
+        input_type='string',
+        label='Email subject',
+        help_text='You can use {event_name} as a placeholder.',
```

Comment:
Maybe extend this helptext a little bit more. Maybe explain that one can also use other object names that are filled in by OpenSlides.

Reply:
The user can only use `{event_name}` here. Do you have a suggestion for this?

Reply:
Ok, no idea.

Reply:
Sorry, no idea.

Reply:
@emanuelschuetze ?

Reply:
I like the current default value and help text. So let us merge this implementation now. We can extend it later.